ilc/server/tailor/configs-injector.js

```diff
@@ -24,53 +24,32 @@ module.exports = class ConfigsInjector {
         const regConf = registryConf.data;
 
         document = document.replace('</body>', this.#wrapWithIgnoreDuringParsing(
-            this.#hideHTMLtoAvoidFOUC(false),
             this.#getSPAConfig(regConf),
             this.#getPolyfill(),
             this.#wrapWithAsyncScriptTag(this.#getClientjsUrl()),
         ) + '</body>');
 
         document = document.replace('</head>', this.#wrapWithIgnoreDuringParsing(
             newrelic.getBrowserTimingHeader(),
-            this.#hideHTMLtoAvoidFOUC(),
         ) + '</head>');
 
-        const scriptRefs = await this.#getRouteScriptRefs(reqUrl);
-
+        const routeAssets = await this.#getRouteAssets(reqUrl);
+        
         document = document.replace('<head>', '<head>' + this.#wrapWithIgnoreDuringParsing(
-            ...scriptRefs.map(this.#wrapWithLinkToPreloadScript),
+            ...routeAssets.scriptLinks,
+            ...routeAssets.stylesheetLinks,
```

Comment:
Let's make stylesheet links to be higher then script preloads

Reply:
Moved